docs/developer-docs/latest/development/export-import.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Data export-import transfer system - Strapi Developer Docs
```

Comment:
I'm not sure how I feel about "Data export-import transfer system". It feels like it should all be together, either export-import-transfer or maybe export/import/transfer. Or something simpler altogether 😆

@marcoautiero should we discuss the official developer release name of this feature?

Reply:
Renamed.